scraping/spiders/canada_immigration_services.py

```diff
@@ -0,0 +1,104 @@
+import re
+import logging
+
+import scrapy
+import hjson
+
+from config.constants import DEFAULT_CATEGORY
+
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
Why is this file here? Are you combining 2 PRs in one?

Reply:
I actually thought that it was merged. Is it okay to merge it?

Reply:
Yes, you can merge it

Reply:
Cool, thanks!